app/src/main/java/com/github/se/orator/ui/navigation/NavigationActions.kt

```diff
@@ -77,6 +100,7 @@ open class NavigationActions(
   open fun navigateTo(destination: TopLevelDestination) {
 
     navController.navigate(destination.route) {
+      Log.d("inside of navigationActions!", "route is: ${destination.route}")
```

Comment:
I would suggest either deleting this if it has a debugging purpose, or having a tag specifying the class or service issuing the log entry.
```suggestion
      Log.d("NavigationActions", "Navigating to ${destination.route}")
```